migrations/1695024779_add-unique-device-tokens.sql

```diff
@@ -0,0 +1,11 @@
+BEGIN;
```

Comment:
Actually I don't think we need to make a transaction ourselves. I think sqlx will do this for us automatically: https://github.com/launchbadge/sqlx/blob/35ba8e2b93f085bbd3b40d31c2f7b01a9911a9de/sqlx-postgres/src/migrate.rs#L241

Reply:
Thanks for catching it! I'll remove the transaction from the file.